truelearn/utils/metrics/_base.py

```diff
@@ -1,59 +1,7 @@
-from typing import Iterable
-
 from sklearn import metrics
 
 
-def get_precision_score(
-    pred_labels: Iterable[bool], act_labels: Iterable[bool]
-) -> float:
-    """Get the precision score of the prediction.
-
-    Args:
-        pred_labels: An iterable of predicted labels
-        act_labels: An iterable of actual labels
-
-    Returns:
-        The precision score.
-    """
-    return float(metrics.precision_score(pred_labels, act_labels))
-
-
-def get_recall_score(pred_labels: Iterable[bool], act_labels: Iterable[bool]) -> float:
-    """Get the recall score of the prediction.
-
-    Args:
-        pred_labels: An iterable of predicted labels
-        act_labels: An iterable of actual labels
-
-    Returns:
-        The recall score.
-    """
-    return float(metrics.recall_score(pred_labels, act_labels))
-
-
-def get_accuracy_score(
-    pred_labels: Iterable[bool], act_labels: Iterable[bool]
-) -> float:
-    """Get the accuracy score of the prediction.
-
-    Args:
-        pred_labels: An iterable of predicted labels
-        act_labels: An iterable of actual labels
-
-    Returns:
-        The accuracy score.
-    """
-    return float(metrics.accuracy_score(pred_labels, act_labels))
-
-
-def get_f1_score(pred_labels: Iterable[bool], act_labels: Iterable[bool]) -> float:
-    """Get the f1 score of the prediction.
-
-    Args:
-        pred_labels: An iterable of predicted labels
-        act_labels: An iterable of actual labels
-
-    Returns:
-        The f1 score.
-    """
-    return float(metrics.f1_score(pred_labels, act_labels))
+get_precision_score = metrics.precision_score
```

Comment:
Will we still be able to get the docstrings for this?

Reply:
It works, but it comes with some warnings.

Reply:
It looks like this:

<details>
<summary>picture here</summary>

![image](https://user-images.githubusercontent.com/45459572/222948809-8373eeef-f760-45e4-bb53-2572f34b4c1d.png)

</details>

Reply:
I guess it's because the docstring references something in scikit learn module

```
TrueLearn-python-library/truelearn/utils/metrics/__init__.py:docstring of sklearn.metrics._classification.accuracy_score:1: WARNING: undefined label: 'accuracy_score'
TrueLearn-python-library/truelearn/utils/metrics/__init__.py:docstring of sklearn.metrics._classification.accuracy_score:5: WARNING: undefined label: 'precision_recall_f_measure_metrics'
TrueLearn-python-library/truelearn/utils/metrics/__init__.py:docstring of sklearn.metrics._classification.accuracy_score:1: WARNING: undefined label: 'precision_recall_f_measure_metrics'
TrueLearn-python-library/truelearn/utils/metrics/__init__.py:docstring of sklearn.metrics._classification.accuracy_score:1: WARNING: undefined label: 'precision_recall_f_measure_metrics'
```

Reply:
I will restore it to its original form.

Reply:
> I will restore it to its original form.

This is not with the new way of generating the docs though, I'll check how it works with #31.